tests/framework/results/test_context.py

```diff
@@ -447,28 +447,41 @@ def test__bad_aggregator_stratification():
             print(result)
 
 
-@pytest.mark.parametrize("pop_filter", ['familiar=="spaghetti_yeti"', ""])
+@pytest.mark.parametrize(
+    "pop_filter", ['familiar=="spaghetti_yeti"', 'familiar=="hufflepuff"', ""]
```

Comment:
Wait, `familiar=="hufflepuff"`? Shouldn't this be `house=="hufflepuff"`? You are likely filtering out the entire population in this filter as well.